src/schema.rs

```diff
@@ -459,6 +470,18 @@ impl Schema {
                 "map" => Schema::parse_map(complex),
                 "fixed" => Schema::parse_fixed(complex),
                 "bytes" => Schema::parse_bytes(complex),
+                "int" => match complex.get("connect.name") {
+                    Some(name)
+                        if name == "io.debezium.time.Date"
+                            || name == "org.apache.kafka.connect.data.Date" =>
```

Comment:
Urgh, annoying that Debezium has decided not to set `logicalType` with `io.debezium.time.Date`. (It does set it for `org.apache.kafka.connect.data.Date`.) This is fine because it's just our code, but, ugh! So much for interop.

Reply:
yeah it is dumb 😢